gp-ant-task/example/build.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations" 
```

Comment:
This is an example project. I'd suggest you to change the project name to "Example Project"
default target is not required. We may simple remove it from <project..>